addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -588,11 +594,14 @@
 
         <t t-foreach="category_data" t-as="category">
             <div class="mb-2" t-if="(category['slides'] or channel.can_publish) and (search_category and search_category.id == category['id'] or not search_category)">
-                <div class="d-flex align-items-center justify-content-between border-bottom pb-2 mb-3" t-if="category['name'] != 'Uncategorized'">
-                    <t t-set="is_empty_editable" t-value="not category['slides'] and channel.can_publish"/>
+                <t t-set="is_empty_editable" t-value="not category['slides'] and channel.can_publish"/>
+                <div class="d-flex align-items-center justify-content-between border-bottom pb-2 mb-3" t-if="category['name'] != 'Uncategorized' and query_string != '?search_uncategorized=1'">
                     <h5 t-attf-class="m-0 #{'text-muted' if is_empty_editable else ''}"><t t-esc="category['name']"/></h5>
                     <a t-if="category['id'] and not is_empty_editable" t-att-href="'/slides/%s/category/%s' % (slug(channel), category['slug_name'])">View all</a>
-                    <a t-elif="not is_empty_editable" t-att-href="'/slides/%s?uncategorized=1' % (slug(channel))">View all</a>
+                </div>
+                <div class="d-flex align-items-center justify-content-between border-bottom pb-2 mb-3" t-if="category['name'] == 'Uncategorized' and len(category['slides']) > 0 and len(category_data) > 1">
+                    <h5 t-attf-class="m-0 text-muted"><t t-esc="category['name']"/></h5>
+                    <a t-att-href="'/slides/%s?uncategorized=1' % (slug(channel))">View all</a>
```

Comment:
The fact that we have a pager for all categories except the "Unbcategorized" slides is not really consistent behavior.

Is there any way we could easily implement it?

Side note: pager is broken and displayed twice.
But it will have to be fixed in a separate task, targeting the right stable branch.
Don't worry about it here.

Reply:
I reintroduced the previous comportment but with the "View All" available for the uncategorized section. The pager wasn't available at all for uncategorized slides, so I had to made some changes to the route and to the view.